crates/wasm_interp/src/instance.rs

```diff
@@ -1076,9 +1107,28 @@ impl<'a> Instance<'a> {
                 let arg1 = self.value_stack.pop_f32();
                 self.value_stack.push(Value::F32(arg1 / arg2));
             }
-            F32MIN => todo!("{:?} @ {:#x}", op_code, file_offset),
-            F32MAX => todo!("{:?} @ {:#x}", op_code, file_offset),
-            F32COPYSIGN => todo!("{:?} @ {:#x}", op_code, file_offset),
+            F32MIN => {
+                let arg2 = self.value_stack.pop_f32();
+                let arg1 = self.value_stack.pop_f32();
+                let result = if arg1 < arg2 { arg1 } else { arg2 };
+                self.value_stack.push(Value::F32(result));
+            }
+            F32MAX => {
+                let arg2 = self.value_stack.pop_f32();
+                let arg1 = self.value_stack.pop_f32();
+                let result = if arg1 > arg2 { arg1 } else { arg2 };
+                self.value_stack.push(Value::F32(result));
```

Comment:
did you check that this works correctly with NaN? There is a https://doc.rust-lang.org/std/primitive.f64.html#method.max function that considers NaN small, and then an unstable `maximum` function that considers NaN to be big. 

based on https://github.com/WebAssembly/design/issues/214 we should pick the version that "maps to a hardware instruction", so I'd guess normal rust `>` is correct?

Reply:
I didn't check, no! To be honest I've been slightly loose with NaN in general! 😬 

Here's the spec but I have no freakin' idea what it says 😅 
https://webassembly.github.io/spec/core/exec/numerics.html#xref-exec-numerics-op-fmax-mathrm-fmax-n-z-1-z-2

For now I was just getting the interpreter to a certain stage without really getting into NaN issues. But once we start using this interpreter for tests, it will probably force us to debug stuff like this. Might need to implement an isNaN-checking test helper and do a round of fixups on the float ops in general.